lng/english.lng.php

```diff
@@ -2049,5 +2049,4 @@
 $lng['panel']['ihave_configured'] = 'I have configured the services';
 $lng['panel']['system_is_configured'] = 'System is already set as configured';
 $lng['panel']['settings_before_configuration'] = 'Please be sure you adjusted the settings prior to configuring the services here';
-$lng['panel']['alternative_cmdline_config'] = 'Alternatively, just run the following command as root-user in you shell to configure the services automatically';
-$lng['tasks']['remove_pdns_domain'] = 'Delete domain %s from PowerDNS database';
```

Comment:
Why did you remove that line? Please re-add. Rest looks fine to me

Reply:
Don't know what happened, but I put it back.